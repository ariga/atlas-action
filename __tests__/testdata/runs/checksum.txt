{
  "Env": {
    "Driver": "mysql",
    "URL": {
      "Scheme": "mysql",
      "Opaque": "",
      "User": {},
      "Host": "localhost:3306",
      "Path": "/test",
      "RawPath": "",
      "ForceQuery": false,
      "RawQuery": "",
      "Fragment": "",
      "RawFragment": "",
      "DSN": "root:pass@tcp(localhost:3306)/test",
      "Schema": "test"
    },
    "Dir": "migrations/"
  },
  "Steps": [
    {
      "Name": "Migration Integrity Check",
      "Text": "File atlas.sum is invalid",
      "Error": "checksum mismatch"
    }
  ],
  "Schema": {},
  "Files": [
    {
      "Name": "atlas.sum",
      "Error": "checksum mismatch"
    }
  ]
}