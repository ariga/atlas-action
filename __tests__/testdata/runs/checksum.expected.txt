<h1>Atlas Lint Report</h1>
Analyzed <strong>migrations/</strong><br><br>
<table><tr><th>Status</th><th>Step</th><th>Result</th><th>Diagnostics</th></tr><tr><td><div align="center"><img src="https://release.ariga.io/images/assets/error.svg" /></div></td><td>Migration Integrity Check</td><td>File atlas.sum is invalid</td><td>checksum mismatch</td></tr><tr><td><div align="center"><img src="https://release.ariga.io/images/assets/special-warning-icon.svg" /></div></td><td colspan="3">Connect your project to <a href="https://auth.ariga.cloud/signup">Atlas Cloud</a> to get more safety checks</td></tr></table>
